app/src/cli/util.ts

```diff
@@ -0,0 +1,71 @@
+import * as Path from 'path'
+
+import stripAnsi = require('strip-ansi')
```

Comment:
This doesn’t seem to work:

```
[at-loader] ./app/src/cli/util.ts:3:28 
    TS2497: Module '"C:/projects/desktop/node_modules/@types/strip-ansi/index"' resolves to a non-module entity and cannot be imported using this construct.
```

The `.d.ts` file uses `export = <foo>`, which requires that you use `import <foo> = require('foo')` to import it.